packages/shared/src/hooks/useShareComment.ts

```diff
@@ -44,7 +30,7 @@ export function useShareComment(
             text: `${post.title}\n${shortUrl}$}`,
           });
           trackEvent(
-            postAnalyticsEvent('share post', post, {
+            postAnalyticsEvent('share comment', post, {
```

Comment:
@idoshamun this hook is for sharing a comment, but the analytics say `share post` - should we make this change to make it more accurate?

Reply:
Yep. Verified all usages to be only for comments.